### PR TITLE
Update en-GB page 2 English language

### PR DESCRIPTION
**Pull request by Twincarb**
Pull Request for Issue # .
#### Summary of Changes

Updated en-GB install language
#### Testing Instructions

code review
The first sentence is left as a cliff hanger, "one" completes the sentence. 

If the second sentence is still true the randomly generated prefix would be 3 or 4 alpha numerics, to reflect default being 5 I have adjusted the wording to "for or five" characters long, the last part of the sentence is adjusted for better grammer.

### CODE REVIEW

**Comment by infograf768** [comment]
While you are there, please change b to strong.

**Comment by Twincarb** [comment]
@infograf768 with the extra markup, I am now wondering if the only points that need to be emphasised are as below... the first sentence is already given pre filled with the option to change. Whereas the second sentence is an instruction if the default isn't used.


Choose a table prefix or use the randomly generated one, ideally four or five characters long. <b>It may only contain alphanumeric characters and MUST end in an underscore. Make sure that the prefix chosen is not used by other tables</b>.


Choose a table prefix or use the randomly generated one, ideally four or five characters long. It <b>may only</b> contain alphanumeric characters <b>and MUST end</b> in an underscore. Make sure that the prefix chosen is not used by other tables.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9424">issues.joomla.org/joomla-cms/9424</a>.</sub>

File: installation/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -37,7 +37,7 @@ INSTL_DATABASE_OLD_PROCESS_DESC="Any existing backup tables from former Joomla!
 INSTL_DATABASE_OLD_PROCESS_LABEL="Old Database Process"
 INSTL_DATABASE_PASSWORD_DESC="For site security using a password for the database account is mandatory."
 INSTL_DATABASE_PASSWORD_LABEL="Password"
-INSTL_DATABASE_PREFIX_DESC="Choose a table prefix or use the <b>randomly generated</b>. Ideally, three or four characters long, contain only alphanumeric characters and MUST end in an underscore. <b>Make sure that the prefix chosen is not used by other tables</b>."
+INSTL_DATABASE_PREFIX_DESC="Choose a table prefix or use the <b>randomly generated</b> one. Ideally, four or five characters long, it may only contain alphanumeric characters and MUST end in an underscore. <b>Make sure that the prefix chosen is not used by other tables</b>."
```
Comment (brianteeman) [inline installation/language/en-GB/en-GB.ini]: why did you change it to four or five from three or four

Comment (Twincarb) [inline installation/language/en-GB/en-GB.ini]: @brianteeman I have only ever seen a default of 5 apart from when it used to be jos_ it doesn't make sense to suggest 3 or 4 when the default is now 5.
But that also defeats my logic as well.
I am not aware of the logic behind 5 but if 5 was decided upon for security then a better change would also be to "Ideally five characters or longer,"

Comment (brianteeman) [inline installation/language/en-GB/en-GB.ini]: quite right it does generate one of 5 characters - dont know when it
changed to that but the language file should have been updated at the time
- so how about updating it to
> Ideally, at least four or five characters long,

Comment (Twincarb) [inline installation/language/en-GB/en-GB.ini]: Apart from in the install the only things I could find was " a few characters" in the documentation, if this PR is accepted I will find locations where it refers to a few and put in amendments so it's consistent.
Rereading it the first sentence is a bit wishy the first word being choose, no choice is given. 
Will edit my PR to make more sense.

**Comment by andrepereiradasilva** [comment]
IMHO the strong should only be used if is really really important to call for the end user attention.
Also i think you either uses CAPITALIZATION or you use **strong**, using **CAPITALIZATION WITH STRONG**, in my personal view, is an overkill and not user friendly.

I think bold in the "must" is enough to call the user attention and still be easily read.

BTW why the "may" and the "MUST" ... aren't both "MUST"?

My proposal:
> Create a table prefix or use the randomly generated one. Ideally four or five characters long, it **must** contain only alphanumeric characters and end in an underscore. Also, make sure that the prefix chosen is not already used by other tables.

**Comment by Twincarb** [comment]
Out of the 3 sentences the 2nd and 3rd are the ones that carry an instruction, I am happy to put the PR forward in all lower case and no emphasis, as I suspect the only time it is read is when someone is installing for the first time, and then they are more likely to read each word and stay with the randomly generated one.

@andrepereiradasilva 
The reason for a may and a must come from my work and using legally binding documentation where instructions are formatted in a particular way.
With the "Must" it is an instruction which has something which has to be followed ie it must end in an underscore. 
However "May only" gives a degree of optionality, as in we are not instructing that it must contain both letters and numbers, rather it "may only" contain letters or numbers. it's just semantics for a project like Joomla mind.

Am open to any option mind, as minor as it is the wording could be improved.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9424">issues.joomla.org/joomla-cms/9424</a>.</sub>

**Comment by brianteeman** [comment]
May and must seem correct to me ( as the only non-lawyer in my family :) l

**Comment by andrepereiradasilva** [comment]
> Out of the 3 sentences the 2nd and 3rd are the ones that carry an instruction, I am happy to put the PR forward in all lower case and no emphasis, as I suspect the only time it is read is when someone is installing for the first time, and then they are more likely to read each word and stay with the randomly generated one.

ok.

> The reason for a may and a must come from my work and using legally binding documentation where instructions are formatted in a particular way.
> With the "Must" it is an instruction which has something which has to be followed ie it must end in an underscore. 
> However "May only" gives a degree of optionality, as in we are not instructing that it must contain both letters and numbers, rather it "may only" contain letters or numbers. it's just semantics for a project like Joomla mind.

> May and must seem correct to me ( as the only non-lawyer in my family :) l

2 to 1, let's democracy play. You win :)

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 07446dfdcdbeef8e79a919a2541ce9889f37fefc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9424">issues.joomla.org/joomla-cms/9424</a>.</sub>

**Comment by waader** [comment]
I have tested this item :white_check_mark: successfully on 07446dfdcdbeef8e79a919a2541ce9889f37fefc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9424">issues.joomla.org/joomla-cms/9424</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9424">issues.joomla.org/joomla-cms/9424</a>.</sub>
